extensions/src/ACRE2Arma/arma2ts/CallExt_DllMain.cpp

```diff
@@ -16,13 +16,13 @@
 
 // trim from start
 static inline std::string &ltrim(std::string &s) {
-    s.erase(s.begin(), std::find_if(s.begin(), s.end(), std::not1(std::ptr_fun<int, int>(std::isspace))));
+    s.erase(s.begin(), std::find_if(s.begin(), s.end(), std::not1(std::ptr_fun<int32_t, int32_t>(std::isspace))));
```

Comment:
This should really be converted to using a lambda. std::ptr_fun has been deprecated in C++11. and std::not1 has been deprecated in c++17.

Reply:
Here is example code for ltrim/rtrim https://github.com/dedmen/armake/blob/cpp/src/utils.cpp#L251
Also supporting tabs.